config/config.go

```diff
@@ -458,10 +458,20 @@ func (p *GatewayConfig) String() string {
 	return "<config object>"
 }
 
-// Var returns the gateway config itself as an expvar
+// Var returns a clone of the gateway config with secrets being redacted
 func (p *GatewayConfig) Var() expvar.Var {
 	return expvar.Func(func() interface{} {
-		return p
+		redacted := "redacted"
+		safeExport := p
```

Comment:
Is that a deep clone?

Reply:
Yeah, I have a runable example here: https://play.golang.org/p/Mm7_aeO6Y5C